gen/build_deploy/bash.py

```diff
@@ -638,9 +638,23 @@ def validate_custom_check_bins_dir(custom_check_bins_dir):
 def generate(gen_out, output_dir):
     print("Generating Bash configuration files for DC/OS")
     make_bash(gen_out)
+    make_powershell(gen_out)
     util.do_bundle_onprem(gen_out, output_dir)
 
 
+def make_powershell(gen_out) -> None:
+    """Build powershell deployment script."""
+    # Populate the powershell script to Bootstrap for further deployment of Windows agent
+    powershell_script_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'powershell/dcos_install.ps1')
```

Comment:
what about the `https://github.com/dcos/dcos/blob/master/gen/build_deploy/powershell/install_choco.ps1` file, isn't this necessary as well?